doc/admin-guide/plugins/wasm.en.rst

```diff
@@ -114,5 +113,6 @@ The plugin will not support the following functionality as specified in Proxy-Wa
 
 * Getting and setting trailer request and response header
 * Getting and setting HTTP/2 frame meta data
+* Support asynchronous request call in the start handler function of the transaction lifecycle
```

Comment:
Should this be in limitations? I thought this is a new feature but adding it here implies it is not supported

Reply:
Yes. This is a new feature but there is no easy way for us to support async request inside the "start" handler function of the plugin lifecycle. 

Let me reword this a bit.